lib/modules/dosomething/dosomething_rogue/dosomething_rogue.module

```diff
@@ -226,7 +239,8 @@ function dosomething_rogue_handle_failure($values, $response = NULL, $failed_tas
   // Save fail to a db log so we can easily export.
   if ($values['type'] === 'reportback') {
     watchdog('dosomething_rogue', 'Reportback not migrated to Rogue: northstar_id: !northstar_id, campaign_id: !campaign_id, campaign run_nid: !campaign_run_id.', ['!northstar_id' => $values['northstar_id'], '!campaign_id' => $values['campaign_id'], '!campaign_run_id' => $values['campaign_run_id']], WATCHDOG_ERROR);
-
+  } elseif ($values['type'] === 'signup') {
```

Comment:
I sorta feel like it would clean up this handle failures function if we separated it out the signup logic and the reportback logic. So like maybe make a `dosomething_rogue_handle_signup_failures()` function?